### ISSUE TITLE
please document the paths that this snap will add

### ISSUE THREAD

**Issue by SingingBush** [dlang-snaps/dmd.snap#52]
I'm currently doing https://github.com/intellij-dlanguage/intellij-dlanguage/issues/575 to add support to dmd installations via this snap. Please can you confirm the following paths are correct:

Path | description | Official .rpm equivalent
-- | -- | --
`/snap/dmd/current/bin/dmd` | dmd binary | `/usr/bin/dmd`
`/snap/phobos/current/import/phobos`  ?? | phobos | `/usr/include/dmd/phobos`
`/snap/druntime/current/import/druntime` ?? | druntime | `/usr/include/dmd/druntime/import`
do you even provide this? | html docs | `/usr/share/dmd/html/d`

Ideally this information should be readily available in the README page

**Comment by WebDrake** [comment]
Hey, thanks for implementing this support, and for reaching out!  The binary, druntime and phobos paths are correct, yes.  HTML docs are not included right now: is that something you need, or just a nice-to-have?

Note that the snap-packaged dmd is invoked via `/snap/bin/dmd` not by directly invoking the binary in `/snap/dmd/current/bin/`.  Technically that is a symlink to `/usr/bin/snap` which is used to launch all snap commands.  So if what you need to configure is the command to run the compiler, that's the one to use.

If it's relevant, you also have `rdmd`, `demangle`, and `dustmite` all in there.

Thanks for the suggestion on how to improve the README, I'll try to update things there.

**Comment by SingingBush** [comment]
Thanks for confirming, I'll update the DMD path. Is dub included? 
The documentation would be nice to have but the Intellij plugin doesn't make use of it yet. I'd like to be able to display documentation directly in the IDE but getting it working with Intellij SDK had been troublesome.

**Comment by WebDrake** [comment]
No, dub is in a separate snap package: https://github.com/dlang-snaps/dub.snap.
